test/framework/src/main/java/org/elasticsearch/test/rest/support/Features.java

```diff
@@ -34,8 +34,8 @@
  */
 public final class Features {
 
-    private static final List<String> SUPPORTED =
-            Arrays.asList("stash_in_path", "groovy_scripting", "headers", "embedded_stash_key", "yaml");
+    private static final List<String> SUPPORTED = Arrays.asList("stash_in_path", "groovy_scripting", "headers", "embedded_stash_key",
```

Comment:
Can you just break these all onto one line per? This was also changed last week and if that was done then, the change set would just be the additional line.